### ISSUE TITLE
Selector interpolation broken in attributes

### ISSUE THREAD

**Issue by rjgotten** [dotless/dotless#318]
The dotless compiler is ignoring variable interpolation inside attribute selectors:

Given a function and call:
````css
.generate(@type) {
  &[data-type="@{type}"] {
    // ...
  }
}
.foo {
  .generate(~"bar");
  .generate(~"baz");
}
````

Expected output (following the official LESS compiler):
````css
.foo[data-type="bar"] {
  // ...
}
.foo[data-type="baz"] {
  // ...
}
````

Actual output (of the dotless compiler):
````css
.foo[data-type="@{type}"] {
  // ...
}
.foo[data-type="@{type}"] {
  // ...
}
````

Would be nice to see this fixed as it basically ruins any attempt at a data-* driven approach of writing the selectors for common material such as named icons in a spritesheet.

**Comment by rytmis** [comment]
PR #424 fixes this. Issue #350 is a duplicate.

**Comment by rytmis** [comment]
Fixed in 39a8214
